includes/traits/llms-trait-certificate-achievement-template-post-list-table.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * LifterLMS Certificate/Achievement Templates Post List Table trait
+ *
+ * @package LifterLMS/Traits
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * LifterLMS Certificate/Achievement Templates Post List Table trait.
+ *
+ * @since [version]
+ */
+trait LLMS_Trait_Certificate_Achievement_Templates_Post_List_Table {
+
+	/**
+	 * Array of supported engagement templates post types.
+	 *
+	 * @var array
+	 */
+	private $post_types = array(
+		'llms_certificate',
+		'llms_achievement',
+	);
+
+	/**
+	 * Add post row actions filter callback.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function certificate_achievement_template_row_actions() {
```

Comment:
```suggestion
	protected function award_template_row_actions() {
```
This doesn't need to be public, does it?

And using generic "award" rather than "certificate_achievement"

Reply:
you're right, but then I want it to be private. :D

Reply:
Well, it can't be.

Reply:
ahah yeah right